### ISSUE TITLE
ValueError: invalid literal for int() with base 10: ''

### ISSUE THREAD

**Issue by georf** [brokkr/poca#102]
I think this error is on podcast owner site, but there I can not do anything :/ You are my friend :+1: 
```
* Url of subscription? http://www.ndr.de/info/podcast3010.xml

Traceback (most recent call last):
  File "/usr/local/bin/poca-subscribe", line 44, in <module>
    main()
  File "/usr/local/bin/poca-subscribe", line 26, in main
    sub_dic, sub_category = poca.subscribe.user_input_add_sub()
  File "/usr/local/lib/python3.4/dist-packages/poca/subscribe.py", line 121, in user_input_add_sub
    url_stats.print_stats()
  File "/usr/local/lib/python3.4/dist-packages/poca/feedstats.py", line 40, in print_stats
    self.set_lhs()
  File "/usr/local/lib/python3.4/dist-packages/poca/feedstats.py", line 60, in set_lhs
    lhs['avg_duration'] = self.get_avg_duration()
  File "/usr/local/lib/python3.4/dist-packages/poca/feedstats.py", line 91, in get_avg_duration
    durations = list(map(self.itunes2seconds, duration_entries))
  File "/usr/local/lib/python3.4/dist-packages/poca/feedstats.py", line 108, in itunes2seconds
    seconds += pow(60, index) * int(entry)
ValueError: invalid literal for int() with base 10: ''
```

**Comment by brokkr** [comment]
One of the entries had a fourth time number after hours, minutes and seconds. Very strange. I have to say, my faith in German genau'ness and korrekt'ness is taking a beating with all these errors :D But it's good to catch them so thanks once again.

**Comment by brokkr** [comment]
Oh and the fix is in beta9 though #101 remains unsolved for now. I'll take a look at it again tomorrow with fresh eyes.
